### PR TITLE
[CARBONDATA-4240]: Added missing properties on the configurations page

### PR DESCRIPTION
**Pull request by pratyakshsharma**
### Why is this PR needed?
 
 Few properties which were not present on configurations page but are user facing properties have been added.
 ### What changes were proposed in this PR?

 Addition of missing properties   
 ### Does this PR introduce any user interface change?
 - No

 ### Is any new testcase added?
 - No

### CODE REVIEW

File: docs/configuration-parameters.md [hunk 1]
```diff
@@ -52,6 +52,18 @@ This section provides the details of all the configurations required for the Car
 | carbon.trash.retention.days | 7 | This parameter specifies the number of days after which the timestamp based subdirectories are expired in the trash folder. Allowed Min value = 0, Allowed Max Value = 365 days|
 | carbon.clean.file.force.allowed | false | This parameter specifies if the clean files operation with force option is allowed or not.|
 | carbon.cdc.minmax.pruning.enabled | false | This parameter defines whether the min max pruning to be performed on the target table based on the source data. It will be useful when data is not sparse across target table which results in better pruning.|
+| carbon.storelocation | (none) | This parameter defines the path on DFS where carbondata files and metadata will be stored. |
```
Comment (MarvinLitt) [inline docs/configuration-parameters.md:55]: carbon.storelocation is need to configure now? does it change to spark.sql.warehouse.dir ?
@QiangCai @kumarvishal09

File: docs/configuration-parameters.md [hunk 2]
```diff
@@ -52,6 +52,18 @@ This section provides the details of all the configurations required for the Car
...
+| spark.deploy.zookeeper.url | (none) | The zookeeper url to connect to for using zookeeper based locking |
+| carbon.data.file.version | V3 | This specifies carbondata file format version. Carbondata file format has evolved with time from V1 to V3 in terms of metadata storage and IO level pruning capabilities. You can find more details [here](https://carbondata.apache.org/file-structure-of-carbondata.html#carbondata-file-format). |
+| spark.carbon.hive.schema.store | false | Carbondata currently supports 2 different types of metastores for storing schemas. This property specifies if Hive metastore is to be used for storing and retrieving table schemas |
+| spark.carbon.sessionstate.classname | `org.apache.spark.sql.hive.CarbonInMemorySessionStateBuilder` | This parameter determines the implementation of carbon session state to override sql parser and for adding strategies. Currently 2 different implementations are provided out of the box - one implements in memory session state and second implements Hive aware session state. |
```
Comment (MarvinLitt) [inline docs/configuration-parameters.md:65]: org.apache.spark.sql.hive.CarbonInMemorySessionStateBuilder is not a commonly used value, give a commonly used value

File: docs/configuration-parameters.md (hunk 1 above)
Comment (pratyakshsharma) [reply docs/configuration-parameters.md]: Taken care of.

File: docs/configuration-parameters.md (hunk 2 above)
Comment (pratyakshsharma) [reply docs/configuration-parameters.md]: This is internally taken care of. Hence removed the property itself.

**Comment by pratyakshsharma** [comment]
@MarvinLitt Please take a look.

**Comment by CarbonDataQA2** [comment]
Build Success with Spark 3.1, Please check CI http://121.244.95.60:12602/job/ApacheCarbon_PR_Builder_3.1/313/

**Comment by CarbonDataQA2** [comment]
Build Success with Spark 2.4.5, Please check CI http://121.244.95.60:12602/job/ApacheCarbon_PR_Builder_2.4.5/4167/

**Comment by CarbonDataQA2** [comment]
Build Success with Spark 2.3.4, Please check CI http://121.244.95.60:12602/job/ApacheCarbonPRBuilder2.3/5911/

**Comment by pratyakshsharma** [comment]
@akashrn5 @Indhumathi27 Can we merge this?

**Comment by CarbonDataQA2** [comment]
Build Failed  with Spark 3.1, Please check CI http://121.244.95.60:12602/job/ApacheCarbon_PR_Builder_3.1/462/

**Comment by pratyakshsharma** [comment]
retest this please

**Comment by pratyakshsharma** [comment]
This should be good to land now @vikramahuja1001 @akashrn5

**Comment by CarbonDataQA2** [comment]
Build Success with Spark 2.3.4, Please check CI http://121.244.95.60:12602/job/ApacheCarbonPRBuilder2.3/6081/

**Pull request closed without merging**